### PR TITLE
Added full name to emergency contact updates as well as insurance updates in admin logs (updated)

### PR DESCRIPTION
**Pull request by JohnCox2211**
No more duplicates are allowed in DB for emergency contacts and insurance info. They are also  properly
displayed when changed in the admin logs now.
#1124

### CODE REVIEW

**Comment by JohnCox2211** [comment]
This issue is a continuation and fix for merge errors that broke this fix:  https://github.com/BCStudentSoftwareDevTeam/celts/pull/1212

File: app/controllers/main/routes.py [hunk 1]
```diff
@@ -188,19 +186,19 @@ def viewUsersProfile(username):
                                 totalSustainedEngagements = totalSustainedEngagements,
                             )
     abort(403)
-
-@main_bp.route('/profile/<username>/emergencyContact', methods=['GET', 'POST'])
+###############################EMERGENCY CONTACT######################################
```
Comment (WackyWeaver) [inline app/controllers/main/routes.py:189]: This seemed to be for finding where you were working in the page. It's much different than the other comments throughout. Maybe have just one #

File: app/controllers/main/routes.py [hunk 2]
```diff
@@ -212,16 +210,18 @@ def emergencyContactInfo(username):
...
+
+        createActivityLog(f"{g.current_user.fullName}  updated {contactInfo.user.fullName}'s emergency contact information.")
         flash('Emergency contact information saved successfully!', 'success') 
         
         if request.args.get('action') == 'exit':
             return redirect (f"/profile/{username}")
         else:
             return redirect (f"/profile/{username}/insuranceInfo")
-
+###############################INSURANCE INFO######################################
```
Comment (WackyWeaver) [inline app/controllers/main/routes.py:224]: same here

**Comment by esw0624** [comment]
I believe this is ready for a merge.

**Review by BrianRamsay — CHANGES_REQUESTED** [review]
contactInfo will be empty if there is no emergency contact for that user yet (which will fail later when we create the activity log)
Logic for when to create the contact info doesn't work

**Comment by eunsungw** [comment]
Made a new branch for this issue, so closing this branch and PR.

**Pull request closed without merging**